frontend/src/pages/Setup/SetupRouter/AlertsSetup.tsx

```diff
@@ -515,25 +518,65 @@ const AlertPicker = function ({
 				)
 			})}
 
-			<Box
-				mt="8"
-				alignItems="center"
-				display="flex"
-				gap="4"
-				color="weak"
-				flexWrap="nowrap"
-			>
-				<Text size="medium" color="moderate" lines="1">
-					We've prefilled the alert configurations based on typical
-					use-cases.{' '}
-				</Text>
-				<TextLink href="/alerts">
-					<Box alignItems="center" display="flex" gap="2">
-						<Text size="medium">Go to alerts</Text>
-						<IconSolidCheveronRight />
-					</Box>
-				</TextLink>
-			</Box>
+			<Stack gap="4" mt="24">
+				<Box>
+					<Button
+						trackingId="save-alerts"
+						kind="primary"
+						size="small"
+						emphasis="high"
+						iconRight={<IconSolidCheveronRight />}
+						loading={createLoading.current}
+						disabled={createLoading.current}
+						onClick={async () => {
```

Comment:
This is a lot for an inline onClick. Consider extracting into a separate function.

Reply:
good idea! cleaning this up